packages/shared/modules/wallet-transfer/components/wallet-transfer-form.tsx

```diff
@@ -22,10 +22,19 @@ const getWalletsTo = (
   return wallets.filter(wallet => wallet.id !== sourceId);
 };
 
+const getSelectedWallet = (
+  wallets: DeepReadonly<WalletData[]>,
+  currentWalletId: string
+): WalletData => {
+  return (
+    wallets.find(wallet => wallet.id === currentWalletId) || ({} as WalletData)
```

Comment:
Minimize to 
`WalletData => wallets.find(wallet ... `

Reply:
done